exercises/crypto-square/description.md

```diff
@@ -49,19 +53,19 @@ with `c` chunks of `r` length, separated by spaces. For phrases that are
 chunks with a single trailing space.
 
 ```text
-imtgdvs fearwer mayoogo anouuio ntnnlvt wttddes aohghn  sseoau 
+"imtgdvs fearwer mayoogo anouuio ntnnlvt wttddes aohghn  sseoau "
 ```
 
 Notice that were we to stack these, we could visually decode the
 cyphertext back in to the original message:
 
 ```text
-imtgdvs
-fearwer
-mayoogo
-anouuio
-ntnnlvt
-wttddes
-aohghn
-sseoau
+"imtgdvs"
+"fearwer"
+"mayoogo"
+"anouuio"
+"ntnnlvt"
+"wttddes"
+"aohghn"
```

Comment:
Should the trailing space be still present here?

Reply:
@Insti long time no see!  Welcome back.
> Should the trailing space be still present here?

I wasn't sure.  There was no white space there to begin with.  #1208 makes no mention of adding white space.  So I thought it best for the first commit to not add any white space.  I thought it best to leave it open for discussion.  I have no issues with adding white space if the consensus is it is needed.

Reply:
(Thanks, I'm not sure if I'm back back but I'm here now :smile: )

>  There was no white space there to begin with.

I think that this file has been a casualty of of some of the trailing whitespace clearing attempts which has removed them. 

Either the appropriate trailing whitespace should be added or the quotes should not be included in this section. The trailing whitspace here is a pretty important part of the exercise.